d15331b25242c5a5d80118b139e7a27bb95076bb
apollo-client/src/test/java/com/ctrip/framework/apollo/spring/spi/ApolloConfigRegistrarHelperTest.java
--- 
+++ 
@@ -1,38 +1,38 @@
 /*
- * Copyright 2021 Apollo Authors
+ * Copyright 2022 Apollo Authors
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
  * http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
  */
 package com.ctrip.framework.apollo.spring.spi;
 
 import static org.springframework.test.util.AssertionErrors.assertEquals;
 
 import com.ctrip.framework.apollo.spring.annotation.ApolloConfigRegistrar;
 import java.lang.reflect.Field;
 import org.junit.Test;
 import org.springframework.util.ReflectionUtils;
 
 public class ApolloConfigRegistrarHelperTest {
 
   @Test
   public void testHelperLoadingOrder() {
     ApolloConfigRegistrar apolloConfigRegistrar = new ApolloConfigRegistrar();
 
     Field field = ReflectionUtils.findField(ApolloConfigRegistrar.class, "helper");
     ReflectionUtils.makeAccessible(field);
     Object helper = ReflectionUtils.getField(field, apolloConfigRegistrar);
 
     assertEquals("helper is not TestRegistrarHelper instance", TestRegistrarHelper.class, helper.getClass());
   }
 }